readthedocs/settings/base.py

```diff
@@ -101,6 +101,16 @@ class CommunityBaseSettings(Settings):
     RTD_MAX_CONCURRENT_BUILDS = 4
     RTD_BUILD_STATUS_API_NAME = 'docs/readthedocs'
 
+    # String contents to store in each virtualenv/pip.conf, useful to set a
+    # custom index-url.
+    RTD_PIP_CONF = ""
```

Comment:
It may be good to add a `BUILDER` prefix as well here, since this variable is only for builders: `RTD_BUILDER_PIP_CONF`